### ISSUE TITLE
tfrnnlm install

### ISSUE THREAD

**Issue by kunazure** [kaldi-asr/kaldi#4121]
Hello,  I'm using tfrnnlm in kaldi to do my rescoring works, I try to use the scripts in `tools/extras/install_tensorflow_cc.sh` and `tools/extras/install_tensorflow_py.sh`  to install my tensorflow environment, but the version in scripts (1.12 and 1.2 with bazel 0.15) do not match the version in `src/tfrnnlm` and `steps/tfrnnlm/*.py` (2.0.0 with bazel 0.26.1). I install the tensorflow 2.0.0 by myself, but I think it might be better to update the installation script,  so I made this suggestion, hope you can refer to it. 

`BAZEL_VERSION=0.15.0`
`GIT checkout r1.12`

**Comment by danpovey** [comment]
Can you please use git log to check who last modified those scripts, and tag them in this issue?
you could make a PR if you know what change you are proposing.

**Comment by kunazure** [comment]
> Can you please use git log to check who last modified those scripts, and tag them in this issue?
> you could make a PR if you know what change you are proposing.
`[build,src] Upgrade TensorFlow RNN to 2.0 (#3771)

* Improve CMake build script

* Builds or ignore CUDA objects based on availibility of CUDA tools.
* Builds C++ binaries of tfrnnlm if `-DTENSORFLOW_DIR=...` is provided.
* Select Apple BLAS by default when building on MacOS.
* Enable CTest for tests (run `ctest` or `make test`).

Typical build on Linux with CUDA+MKL:

```
export MKLROOT
cmake -DMATHLIB=MKL <cmakefile>
make
ctest
```

* Update model training script to TensorFlow v2 flavor

* Enable rescore tool to load from SavedModel format

* Update the training script which uses custom loss function

Still tried to mimic original behavior without whole pipeline ready.

* Fix numerical stability of the fast softmax model`

the src  and python file was last modified by @scottcjt, but I don't find  change  about the tensorflow version in the installation scripts.   Due to network limitations, I used a lot of manual operations during 
tensorflow installation, which may not be suitable for script reproduction. So is it possible to add some prompt or update  the installation script to avoid additional installation work? Thank you for your reply.

**Comment by danpovey** [comment]
maybe you could make a PR just with a draft of your changes?

**Comment by kunazure** [comment]
> maybe you could make a PR just with a draft of your changes?

OK, I made a PR just now,  thank you.
